src/import/__snapshots__/pprof.test.ts.snap

```diff
@@ -223,7 +319,11 @@ Object {
       "col": undefined,
       "file": "/usr/local/Cellar/go/1.10.1/libexec/src/runtime/sys_darwin_amd64.s",
       "key": "runtime.usleep:/usr/local/Cellar/go/1.10.1/libexec/src/runtime/sys_darwin_amd64.s:0",
-      "line": 0,
+      "line": Long {
+        "high": 0,
+        "low": 418,
+        "unsigned": false,
+      },
```

Comment:
I'm confused about what's happening here. What JavaScript type is this? I would expect this diff to show a change from `"line": 0` to `"line": 418`, rather than what this is. Is this a `BigInt` type?

Reply:
I have no idea what this is, the code was generated when I regenerated the fixtures 🤷‍♂️ 

typescript isn't my forte so any advice would be appreciated.

Reply:
In protobufjs, we [pull in the `long` module](https://github.com/protobufjs/protobuf.js/blob/56b1e64979dae757b67a21d326e16acee39f2267/src/util/minimal.js#L182). And then this apparently causes protobuf decoding to create `Long` objects instead of JS numbers.

For some reason, parcel.js doesn't seem to pull `long` into the web build, and so this only happens in tests, not in the webapp. I'm not familiar enough with JS build systems to understand how to fix that.